web/ViewModels/TenantMatchListModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿using BricksAndHearts.Database;
+namespace BricksAndHearts.ViewModels;
+
+public class TenantMatchListModel
+{
+    public List<TenantDbModel>? TenantList { get; set; }
+    public PropertyViewModel CurrentProperty { get; set; } = null!;
```

Comment:
This is a bit of an anti pattern here, we are seemingly saying that the `CurrentProperty` is not null, as its not `PropertyViewModel?`, but then we are setting it to be equal to null. Either the type should change to nullable (and then we don't need to instantiate it with null), or we should be instantiating it